### DISCUSSION TITLE
Version 1.1.0 Feedback & Questions

### DISCUSSION THREAD

**Discussion by lesydimitri** [privacyidea/shibboleth-plugin#56] [Q&A] [answered]
Hi,

Thanks for the work on v1.1.0, I can confirm that the new version works with Shibboleth IdP v5 :-)

I did run into an issue installing though: The resources were not automatically copied into my installation, I had to manually copy the CSS, JS, Velocity Template and privacyidea.properties files. After I did that everything worked. 

I tested the new version with some users with multiple tokens: TOTP and email. In this case, the current UI flow is a bit confusing to me. I would have expected the user to be able to choose the token to be used (OTP or email in my case). Instead, the OTP field is immediately displayed and the email token is only requested after submitting an empty OTP field.

Another use case where I would be interested in the "correct" way of implementing it is for users without a token. I have some users in PrivacyIDEA with "passOnNoToken". Currently, these users are still shown the OTP page. Submitting the empty page works, but this flow would definitely confuse end users.

Kind regards
Dimitri Lesy

**Comment by lukasmatusiewicz** [comment, marked as answer]
Hi @lesydimitri ,

you can use send static pass function (with empty pass) to fulfill this scenario (see readme).
And to the resources: sometimes you must enable the module manually after the first try. I put some note about it in the readme.
If you have any other questions let me know here:)

Greets
Lukas Matusiewicz
